nav2_bringup/launch/tb3_simulation_launch.py

```diff
@@ -176,10 +176,13 @@ def generate_launch_description():
         condition=IfCondition(use_simulator),
         cmd=[
             'gzserver',
+            '--verbose',
             '-s',
             'libgazebo_ros_init.so',
             '-s',
             'libgazebo_ros_factory.so',
+            '-e',
+            'dart',
```

Comment:
@ahcorde can you remove all the changes in this file. I was experimenting and forgot to remove the changes.